controls/SV-221584.rb

```diff
@@ -21,4 +21,10 @@
   tag legacy: ["SV-57639","V-44805"]
   tag cci: ["CCI-002605"]
   tag nist: ["SI-2 c"]
+
+  domain_role = command('(Get-Item (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe').'(Default)').VersionInfo.ProductVersion')
```

Comment:
Like this ?

Reply:
Will need to use backslash, `\`, to escape some of the single quotes, `'`, inside the command function. Consider the following line to fix the syntax. 
`domain_role = command('(Get-Item (Get-ItemProperty \'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe\').\'(Default)\').VersionInfo.ProductVersion')`